wpilibc/src/main/native/include/frc/Compressor.h

```diff
@@ -58,37 +57,6 @@ class Compressor : public wpi::Sendable,
   Compressor(Compressor&&) = default;
   Compressor& operator=(Compressor&&) = default;
 
-  /**
-   * Starts closed-loop control. Note that closed loop control is enabled by
-   * default.
-   *
-   * @deprecated Use EnableDigital() instead.
-   */
-  WPI_DEPRECATED("Use EnableDigital() instead")
-  void Start();
-
-  /**
-   * Stops closed-loop control. Note that closed loop control is enabled by
-   * default.
-   *
-   * @deprecated Use Disable() instead.
-   */
-  WPI_DEPRECATED("Use Disable() instead")
-  void Stop();
-
-  /**
-   * Check if compressor output is active.
-   * To (re)enable the compressor use EnableDigital() or EnableAnalog(...).
-   *
-   * @return true if the compressor is on.
-   * @deprecated To avoid confusion in thinking this (re)enables the compressor
-   * use IsEnabled().
-   */
-  WPI_DEPRECATED(
-      "To avoid confusion in thinking this (re)enables the compressor use "
-      "IsEnabled()")
-  bool Enabled() const;
```

Comment:
Enabled() was only deprecated April 8, so hasn't even seen a release with it deprecated yet.